doc/changes/changes_0.1.0.md

```diff
@@ -28,12 +29,13 @@ Code name:
 
 ### Test Dependency Updates
 
-* Added `com.exasol:exasol-testcontainers:3.5.3`
 * Added `com.exasol:hamcrest-resultset-matcher:1.4.0`
 * Added `junit:junit:4.13.2`
+* Added `mysql:mysql-connector-java:8.0.23`
 * Added `org.hamcrest:hamcrest:2.2`
 * Added `org.junit.jupiter:junit-jupiter:5.7.2`
 * Added `org.testcontainers:junit-jupiter:1.15.3`
+* Added `org.testcontainers:mysql:1.15.3`
 * Added `org.testcontainers:postgresql:1.15.3`
 
 ### Plugin Dependency Updates
```

Comment:
we should add a line in `Features` for `#7: Added support for MySQL Virtual Schema installation.` or similar